float32 coord_x
float32 coord_y
float32 coord_ang
---
int32 flag
int32 region
float32 x
float32 y
float32 theta
float32[500] sensors
int32 num_sensors
string sensor
